### PR TITLE
COMMERCE-8731 1

### PR DESCRIPTION
**Pull request by ghost**
https://issues.liferay.com/browse/COMMERCE-8731

### CODE REVIEW

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-shipping-engine-fixed-service/src/main/java/com/liferay/commerce/shipping/engine/fixed/internal/upgrade/CommerceShippingEngineFixedUpgradeStepRegistrator.java [hunk 1]
```diff
@@ -70,6 +71,9 @@ public void register(Registry registry) {
 		registry.register(
 			"2.2.0", "2.3.0", new CommerceShippingFixedOptionUpgradeProcess());
 
+		registry.register(
+			"2.3.0", "2.4.0", AccountEntryShippingOptionRelTable.create());
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-shipping-engine-fixed-service/src/main/java/com/liferay/commerce/shipping/engine/fixed/internal/upgrade/CommerceShippingEngineFixedUpgradeStepRegistrator.java:75]: @CrescenzoRegaLF you're missing the bnd.bnd version change

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-shipping-engine-fixed-service/src/main/java/com/liferay/commerce/shipping/engine/fixed/internal/upgrade/CommerceShippingEngineFixedUpgradeStepRegistrator.java (hunk 1 above)
Comment (CrescenzoRegaLF) [reply modules/apps/commerce/commerce-shipping-engine-fixed-service/src/main/java/com/liferay/commerce/shipping/engine/fixed/internal/upgrade/CommerceShippingEngineFixedUpgradeStepRegistrator.java:75]: I added it in the commit
COMMERCE-8731 regenerate the service

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceOrders.macro [hunk 2]
```diff
@@ -22,32 +22,6 @@ definition {
 		Alert.viewSuccessMessage();
 	}
 
-	macro addTermsAndCondition {
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceOrders.macro:25]: why did you change the macros?

File: modules/apps/commerce/commerce-shipping-engine-fixed-api/bnd.bnd [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Commerce Shipping Engine Fixed API
 Bundle-SymbolicName: com.liferay.commerce.shipping.engine.fixed.api
-Bundle-Version: 11.0.1
+Bundle-Version: 11.1.0
```
Comment (andrea-ale-sbarra) [inline modules/apps/commerce/commerce-shipping-engine-fixed-api/bnd.bnd:3]: this should go in a separate commit

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceOrders.macro (hunk 2 above)
Comment (CrescenzoRegaLF) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceOrders.macro:25]: I don't know, I don't think I did it intentionally

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceOrders.macro (hunk 2 above)
Comment (CrescenzoRegaLF) [reply modules/apps/commerce/commerce-product-test/src/testFunctional/macros/CommerceOrders.macro:25]: i can't find the commit where i would edit the file

**Comment by CrescenzoRegaLF** [comment]
ci:stop

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by CrescenzoRegaLF** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
sent here https://github.com/liferay-commerce/liferay-portal/pull/2428
